azure-pipelines-samples.yml

```diff
@@ -134,18 +357,19 @@ steps:
       createLogFile: true
     condition: succeededOrFailed()
 
-  - task: VSBuild@1
-    displayName: 'Build SqueezenetObjectDetection Sample'
-    inputs:
-      solution: Samples/SqueezeNetObjectDetection/squeezenetobjectdetection.sln
-      vsVersion: 15.0
-      msbuildArgs: '-v:diag /p:OutDir=$(System.DefaultWorkingDirectory)\bin\$(BuildPlatform)\$(BuildConfiguration)\SqueezeNetObjectDetection\'
-      platform: '$(BuildPlatform)'
-      configuration: '$(BuildConfiguration)'
-      clean: true
-      msbuildArchitecture: x64
-      createLogFile: true
-    condition: succeededOrFailed()
+  # TODO: Bug 25927815: SqueezeNetObjectDetection NetCore Sample is failing to be built
```

Comment:
Make sure to add the SqueezeNetObjectDetection test back in when this get's uncommented. Maybe it would be best to add it in comments now?